internal/provider/resources/install/azure/azure.go

```diff
@@ -77,25 +88,41 @@ func (r *Azure) Schema(ctx context.Context, req resource.SchemaRequest, resp *re
 					stringvalidator.RegexMatches(common.UuidRegex, "Azure Directory ID must be a valid UUID"),
 				},
 			},
-			"client_id": schema.StringAttribute{
-				Required:            true,
-				MarkdownDescription: `The Microsoft Azure Service Account Client ID.`,
-				PlanModifiers: []planmodifier.String{
-					stringplanmodifier.RequiresReplace(),
-				},
-				Validators: []validator.String{
-					stringvalidator.RegexMatches(common.UuidRegex, "Azure Client ID must be a valid UUID"),
-				},
-			},
 			"service_account_email": schema.StringAttribute{
 				Computed:            true,
-				MarkdownDescription: `The identity that P0 uses to communicate with your Microsoft Azure organization`,
+				MarkdownDescription: `The service identity email that P0 uses to communicate with your Microsoft Azure organization`,
 			},
 			"service_account_id": schema.StringAttribute{
 				Computed:            true,
-				MarkdownDescription: `The ID of the service account that P0 uses to communicate with your Microsoft Azure organization`,
+				MarkdownDescription: `The service identity ID that P0 uses to communicate with your Microsoft Azure organization`,
+			},
+			"app_name": schema.StringAttribute{
+				Computed:            true,
+				MarkdownDescription: "The name of the Azure application P0 uses to communicate with your Microsoft Azure organization. This name is used to identify the app in the Azure portal.",
+			},
+			"credential_info": schema.SingleNestedAttribute{
+				Computed:            true,
+				MarkdownDescription: "The credential information to setup Azure application federated credentials. This is used to authenticate P0 with your Microsoft Azure organization.",
+				Attributes: map[string]schema.Attribute{
+					"name": schema.StringAttribute{
+						Computed:            true,
+						MarkdownDescription: "The display name of the Azure application federated credential.",
+					},
+					"description": schema.StringAttribute{
+						Computed:            true,
+						MarkdownDescription: "The description of the Azure application federated credential.",
+					},
+					"issuer": schema.StringAttribute{
+						Computed:            true,
+						MarkdownDescription: "The issuer of the Azure application federated credential.",
+					},
+					"audiences": schema.ListAttribute{
+						Computed:            true,
+						ElementType:         types.StringType,
+						MarkdownDescription: "The audience of the Azure application federated credential. This is used to establish a connection between external workload identities",
```

Comment:
```suggestion
						MarkdownDescription: "The audience of the Azure application federated credential. This is used to establish a connection with the P0 service account",
```

Worth letting them know this will just let us in and not others?